### ISSUE TITLE
OpenCore Sanity Check Battery Warning and others...

### ISSUE THREAD

**Issue by cyberfunk** [DominikHackintosh/HP-EliteBook-840-G4-Hackintosh-Big-Sur#7]
Hi Dominik,
Do you use OpenCore Sanity Check. I ran your config.plist through there and it turns up a few errors and warnings. What I did on Catalina is follow all of their corrections and everything still worked fine when I rebooted. You've made significantly more changes with your latest EFI in particular switching to SMCBatteryManager.kext  and that is throwing a danger warning. If you believe in Sanity Check can you run your plist through there and let me know if I should just ignore all of the warnings? 

[](https://opencore.slowgeek.com)

**Comment by DominikHackintosh** [comment]
i don´t believe in those "errors" of sanity check, i tried acpibatterymanager a long time but smc is way better and makes not any kind of troubles for me. just ignore it ;)
